.github/workflows/loki.yaml

```diff
@@ -0,0 +1,32 @@
+name: Push to Loki
+
+on:
+  workflow_run:
+    workflows:
+      - AKS (Azure IPAM)
+      - AKS (BYOCNI)
+      - EKS (tunnel)
+      - EKS (ENI)
+      - External Workloads
+      - GKE
+      - Go
+      - Image CI Build
```

Comment:
Arguably, these two seems to be dev/compilation/rare issue, which PR author needs to address regardless 🤔

Reply:
yeah if it fails it's likely an issue with the pull request itself, but i think it's still useful to push the data anyways. for example, we can see if the workflow gets slower over time 🔍